.github/workflows/ci.yml

```diff
@@ -67,6 +67,10 @@ jobs:
       - name: Check that workflows are up to date
         run: sbt githubWorkflowCheck
 
+      - uses: coursier/setup-action@v1
+        with:
+          apps: scalafmt
```

Comment:
This needed since `RewriteTest."applyUpdate with scalafmt update"` now actually executes `scalafmt`.